cypress/integration/immutableDatabaseTests/notifications.ts

```diff
@@ -0,0 +1,97 @@
+ /*
+ *    Copyright 2019 OICR
```

Comment:
It's 2020, this should maybe also include UCSC. But does it even matter with open-source? That's probably something for other people to look into, just the 2020 bit for now.